server/config/game/specialists.json

```diff
@@ -255,11 +255,7 @@
                 },
                 "special": {
                     "reigniteDeadStar": true,
-                    "reigniteDeadStarNaturalResources": {
-                        "economy": 25,
-                        "industry": 25,
-                        "science": 25
-                    }
+                    "reigniteDeadStarNaturalResources": 25
```

Comment:
Not sure why this needed to change? We may want to keep resources separate as it's more flexible for potentially future specs.